include/lift.hpp

```diff
@@ -17,16 +17,17 @@
 #include <utility>
 #include <tuple>
 
-#define LIFT_THRICE(...)                \
-        noexcept(noexcept(__VA_ARGS__)) \
-        -> decltype(__VA_ARGS__)        \
-        {                               \
-          return __VA_ARGS__;           \
-        }
-
-#define LIFT_FWD(x) std::forward<decltype(x)>(x)
+#define LIFT_FUNCTION(f)                                                  \
+    LIFT_ba7b8453f262e429575e23dcb2192b33(                                \
+        a_ba7b8453f262e429575e23dcb2192b33,                               \
+        (f)(::std::forward<decltype(a_ba7b8453f262e429575e23dcb2192b33)>( \
+            a_ba7b8453f262e429575e23dcb2192b33)...))
```

Comment:
Actually `LIFT(f(<int, int>)` should work, so...

```suggestion
#define LIFT_FUNCTION(...)                                                  \
    LIFT_ba7b8453f262e429575e23dcb2192b33(                                \
        a_ba7b8453f262e429575e23dcb2192b33,                               \
        (__VA_ARGS__)(::std::forward<decltype(a_ba7b8453f262e429575e23dcb2192b33)>( \
            a_ba7b8453f262e429575e23dcb2192b33)...))
```